web-client/src/client.rs

```diff
@@ -0,0 +1,1209 @@
+use std::{
+    cell::{Cell, RefCell},
+    collections::{
+        hash_map::{Entry, HashMap},
+        HashSet,
+    },
+    rc::Rc,
+    str::FromStr,
+    time::Duration,
+};
+
+use futures::{channel::oneshot, future::select, future::Either};
+use futures_util::StreamExt;
+use js_sys::{global, Array, Function, JsString};
+use log::level_filters::LevelFilter;
+use tsify::Tsify;
+use wasm_bindgen::{
+    prelude::{wasm_bindgen, Closure, JsError, JsValue},
+    JsCast,
+};
+use wasm_bindgen_futures::spawn_local;
+use web_sys::MessageEvent;
+
+pub use nimiq::{
+    config::config::ClientConfig,
+    config::config_file::{LogSettings, Seed},
+    extras::{panic::initialize_panic_reporting, web_logging::initialize_web_logging},
+};
```

Comment:
All `nimiq` includes should be together

Reply:
They are. Do you mean I should remove the newline between `nimiq::` and `nimiq_*::` imports?

Reply:
Yes, basically what you already did :+1: